packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -142,4 +142,44 @@ describe('BlocksEditor toolbar', () => {
       },
     ]);
   });
+  it('transforms the selection to a heading when selected and trasform it back to text when selected again', async () => {
+    setup();
+
+    const headingsDropdown = screen.getByRole('combobox');
```

Comment:
is there a more precise check with could do in addition to the role? this would break if another combobox is added

Reply:
unfortunately my combobox doesn't have a label but I can find a better check

Reply:
What do you think if I add an aria-label to the Select

Reply:
I agree that aria-label is the right thing here, but we need a new translation key for it